frontend/interfaces/host.ts

```diff
@@ -203,6 +203,14 @@ export interface IHost {
   device_users?: IDeviceUser[];
   munki?: IMunkiData;
   mdm?: IHostMdmData;
+  // below 2 fields per https://github.com/fleetdm/fleet/issues/9219
```

Comment:
Yeah, I think we are removing `mdm?: IHostMdmData` above. I also did this in my parallel PR, no need for the comment

Reply:
cool. when merging I'll make sure to exclude the comment.